framework/src/modules/chain/transactions/votes.js

```diff
@@ -89,7 +93,7 @@ const updateSenderRoundInformationWithAmountForTransaction = function(
 			.map(vote => vote.slice(1));
 
 		// Votes inside the transaction should not be incase it's an exception, but the existing votes should be updated
-		if (!exceptions.roundVotes.includes(transaction.id)) {
+		if (!(exceptions.roundVotes || []).includes(transaction.id)) {
```

Comment:
exceptions default configuration should set empty array for roundVotes property.